storage/SystemInfo/Makefile.am

```diff
@@ -23,3 +23,5 @@ libsysteminfo_la_SOURCES =				\
 	ProcMounts.cc		ProcMounts.h		\
 	ProcParts.cc		ProcParts.h
 
+pkginclude_HEADERS =		\
```

Comment:
Unfortunately pkgincludedir must also be set to $(includedir)/storage/SystemInfo.